bin/i18n/resources/dashboard/course_content/sync_in.rb

```diff
@@ -204,7 +204,7 @@ def process
                 functions = blocks.xpath("//block[@type=\"procedures_defnoreturn\"]")
                 i18n_strings['function_definitions'] = Hash.new unless functions.empty?
                 functions.each do |function|
-                  name = function.at_xpath('./title[@name="NAME"]')
+                  name = function.at_xpath('./title[@name="NAME"] | ./field[@name="NAME"]')
```

Comment:
https://github.com/code-dot-org/code-dot-org/blob/b27bc3da49beb03f1669b95ce042adf4a800c26a/dashboard/app/models/levels/blockly.rb#L615-L617

Reply:
Do we need to update the test cases too?

Reply:
https://github.com/code-dot-org/code-dot-org/pull/60047/commits/6af1cddfe9d90520e239e8143d22e79cfdb55c6d